### PR TITLE
Categorize and recategorize some plugins to the new "Starts" category

### PR DESCRIPTION
**Pull request by TheGiraffe3**
Makes a new "Starts" category.

### CODE REVIEW

**Comment by zuckung** [comment]
test worked fine.

template.txt needs more changes
line 129 and 139, other categories are like `[Story](%webroot%res/mds/story.md) (%story%)`  
and you have just `[Starts](%webroot%res/md/starts.md)` so the replacement of the plugin amount is missing.

also the categories are in 3 rows of 4 categories each (till now), and you added a 5th to the 3rd row. so for the eye, make it 3 rows of 4 and a 4th row with a single. not sure about the look, maybe 2 rows with 6 and 7 categories.

**Comment by zuckung** [comment]
plugins which add starts:

DevStarts
better.starts
Difficulties
Lampyrid Start
Advanced Starts
Alternate Start Luna
Space Shuttle Start
AuldrStarts

**Comment by zuckung** [comment]
and to add plugin via issue:

.github/ISSUE_TEMPLATE/plugin-submission.yml line 46

**Comment by TheGiraffe3** [comment]
Please co-author zuckung if/when you merge this.

**Comment by zuckung** [comment]
and planet.map doesn't have a start anymore, and it was more story than just a start.
might need some discussion if only pure starts should be in here or plugins which add lots other things and require using the start too.

File: res/pluginlist/planet.map.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 author=zuckung
 website=https://github.com/zuckung/endless-sky-plugins
 directlink=https://github.com/zuckung/endless-sky-plugins/releases/download/Latest/planet.map.zip
-category=Story
+category=Starts
```
Comment (TheGiraffe3) [inline res/pluginlist/planet.map.txt:4]:
```suggestion
category=Story
```

File: res/pluginlist/Galactic Conquest.txt [hunk 2]
```diff
@@ -1,6 +1,6 @@
 author=SpiralCat
 website=https://www.moddb.com/mods/galactic-conquest
 directlink=N/A
-category=story
+category=Status
```
Comment (TheGiraffe3) [inline res/pluginlist/Galactic Conquest.txt:4]:
```suggestion
category=Starts
```

**Review by Hecter94 — CHANGES_REQUESTED** [review]
Can you please remove all of the unrelated changes from this PR? 
You can open them in a new PR if you want to.

**Comment by TheGiraffe3** [comment]
By "unrelated changes" I assume you mean typo fixes?

**Comment by Hecter94** [comment]
> By "unrelated changes" I assume you mean typo fixes?

Anything that isn't part of adding this new category, yes.

File: res/pluginlist/Difficulties.txt [hunk 3]
```diff
@@ -1,6 +1,6 @@
 author=Mielo Symeonidis
 website=N/A
 directlink=N/A
-category=gameplay
+category=Starts
 status=N/A
-description=Adds three diffrent difficulties that take form as the start to a game. Not reccomended for people new to Endless Sky.
+description=Adds three different difficulties that take form as the start to a game. Not recomended for people new to Endless Sky.
```
Comment (TheGiraffe3) [inline res/pluginlist/Difficulties.txt:6]:
```suggestion
description=Adds three diffrent difficulties that take form as the start to a game. Not reccomended for people new to Endless Sky.
```

File: res/pluginlist/Difficulties.txt [hunk 4]
```diff
@@ -1,6 +1,6 @@
 author=Mielo Symeonidis
 website=N/A
 directlink=N/A
-category=gameplay
+category=Starts
 status=N/A
-description=Adds three diffrent difficulties that take form as the start to a game. Not reccomended for people new to Endless Sky.
+description=Adds three diffrent difficulties that take form as the start to a game. Not reccomended for people new to Endless Sky.
```
Comment (TheGiraffe3) [inline res/pluginlist/Difficulties.txt:6]:
```suggestion
description=Adds three diffrent difficulties that take form as the start to a game. Not reccomended for people new to Endless Sky.

```

File: res/pluginlist/better.starts.txt [hunk 5]
```diff
@@ -1,6 +1,6 @@
 author=zuckung
 website=https://github.com/zuckung/endless-sky-plugins
 directlink=https://github.com/zuckung/endless-sky-plugins/releases/download/Latest/better.starts.zip
-category=gameplay
+category=Starts
 status=tested with 0.10.2
-description=Adds several new start options with different ships, background storys, credits and debts. See the readme for details.
\ No newline at end of file
+description=Adds several new start options with different ships, background storys, credits and debts. See the readme for details.
```
Comment (TheGiraffe3) [inline res/pluginlist/better.starts.txt:6]:
```suggestion
description=Adds several new start options with different ships, background storys, credits and debts. See the readme for details.

```

**Pull request merged**